### DISCUSSION TITLE
Where has XYZ repository gone??

### DISCUSSION THREAD

**Discussion by jroxendal** [directus/directus#3127] [Archive]
I can't access the directus/docker repo on GitHub, has it been removed intentionally? It's still being linked to from the readme.

**Comment by sylvainmetayer** [comment]
It seems that multiple repos are gone too : 

- https://github.com/directus/api-next
- https://github.com/directus/oauth2-okta

**Comment by benhaynes** [comment]
In preparation for the release of Directus 9, all of our repositories have been re-organized. All Directus 8 repos have been archived and consolidated within the `v8-archive` repo. This is a LONG and tedious process, and we'll be finding and fixing broken links as they arise.

* `docker` — https://github.com/directus/v8-archive/tree/master/docker
* `api-next` — https://github.com/directus/v9-laravel
    * This is now a private repo to avoid confusion. We had not received any external contributions here, so we've made it private until it's in an Alpha state.
* `oauth2-okta` — https://github.com/directus/v8-archive/tree/master/packages/oauth2-okta

There were 77 repos previously, if you have questions about where a specific one went you can ask here and we'll update this answer. Hopefully everyone understands that every ten years or so a little house cleaning is required. Having been a decade since our last GitHub overhaul, and now that there's the added confusion of switching from PHP to Node, now was a good time. 😄

**Reply by shmc** [reply]
the package "zendframework/zend-db": "dev-directus" is not available. can you make it available so we can continue to build it?

**Reply by miguelsimoes** [reply]
Regarding the same package, are there any changes between the zendframework:zend-db:2.11.0 version and one used on Directus?

**Reply by benhaynes** [reply]
We're working on the Zend DB package now. Yes, there are likely changes between our fork and Zend's... that's why we had to pull it internal. 😉

**Reply by besucher80** [reply]
Hi , is it possible to have a lock at https://github.com/directus/v9-laravel or Could you make it public ? Thanks

**Comment by westhouseit** [comment]
See also: https://github.com/directus/directus/discussions/3146
